### PR TITLE
Expose more minecraft options

### LINKED ISSUE #4766: [stable/minecraft] Chart fails to expose some options in the underlying container image
**Is this a request for help?**: No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Feature request

**Version of Helm and Kubernetes**: Helm 2.7.0, Kubernetes 1.9.4

**Which chart**: stable/minecraft

**What happened**: I went to install the helm chart so I could replace my bare manifest installation with a nice Helm install, but it's missing some env options I utilize

**What you expected to happen**: The chart exposes all the options available in the underlying container

**How to reproduce it** (as minimally and precisely as possible): It's not really a bug - reading through the `values.yaml` for this chart and comparing it to the environment variable options on https://hub.docker.com/r/itzg/minecraft-server/ will reveal what's missing

**Anything else we need to know**: I'll have a PR in with a fix; I wouldn't have bothered with the issue, but figured workflow probably required it :)

### PR DESCRIPTION
**Pull request by mcronce**
**What this PR does / why we need it**:  The current chart is missing some of the options available in [the underlying container image](https://hub.docker.com/r/itzg/minecraft-server) - this PR adds them; at least, I think I got them all

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #4766

**Special notes for your reviewer**:
This was originally submitted in #5036 - it went stale and was closed, though, and I didn't notice until now.

### CODE REVIEW

**Comment by mcronce** [comment]
@paulczar Thank you

File: stable/minecraft/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: minecraft
-version: 0.2.2
+version: 0.2.3
```
Comment (viglesiasce) [inline stable/minecraft/Chart.yaml]: Bump to 0.3.0

**Review by viglesiasce — CHANGES_REQUESTED** [review]

File: stable/minecraft/Chart.yaml (hunk 1 above)
Comment (mcronce) [reply stable/minecraft/Chart.yaml]: Done...waiting on tests to rerun now

**Review by viglesiasce — APPROVED** [review]

**Pull request merged**